src/jdk.management/unix/classes/com/sun/management/internal/OperatingSystemImpl.java

```diff
@@ -93,7 +93,7 @@ public double getContainerCpuLoad() {
                 int totalCPUs = getHostOnlineCpuCount0();
                 int containerCPUs = getAvailableProcessors();
                 // scale the total host load to the actual container cpus
-                hostTicks = hostTicks * containerCPUs / totalCPUs;
+                hostTicks = (long) (hostTicks * (1.0 * containerCPUs / totalCPUs));
```

Comment:
1.0  used to convert the data type to floating point  and  compute the division first

Reply:
> The intent is to ensure floating point arithmetic on the second expression, right? A clearer way to express this may be the following. This avoids the `1.0` constant:
> 
> ```
> hostTicks = (long) (hostTicks * ((double) containerCPUs)/ totalCPUs)
> ```

1.0 used to convert the data type to floating point and compute the division first

Reply:
Right. It seems a strange way to achieve this. Maybe consider this instead?
```
double scaleFactor = ((double)containerCPUs)/totalCPUs;
hostTicks = (long)(hostTicks * scaleFactor);
```